bittensor/commands/stake.py

```diff
@@ -660,13 +662,15 @@ def _run(cli: "bittensor.cli", subtensor: "bittensor.subtensor"):
             if not wallet_utils.is_valid_ss58_address(child):
                 console.print(f":cross_mark:[red] Invalid SS58 address: {child}[/red]")
                 return
-
+        console.print(f"the number of children is {len(children)}")
         if (
             len(children) == 1
         ):  # if only one child, then they have full proportion by default
             cli.config.proportions = 1.0
```

Comment:
The proportions parameter is a float, not a string. Please take a look.
```suggestion
            cli.config.proportions = "1.0"
```